Makefile

```diff
@@ -114,6 +114,8 @@ install: conda
 # pip install git+https://github.com/mskcc/beagle_cli.git@develop
 # pip install -r requirements-cli.txt # <- what happened to this file?
 
+BEAGLE_BACKUP_DB_FOLDER=/juno/work/ci/beagle_backups/
```

Comment:
not sure if this just makes it more complicated than necessary but this could maybe be an env var